tests/features/main.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Testing main endpoint
+
+Scenario: Testing welcoming endpoint
+    Given I have the API endpoint "/"
+    When I send a GET request to the endpoint
```

Comment:
Rework on all you step, to be exactly like leto-modelizer-api

Reply:
done